src/Compilers/CSharp/Portable/Compilation/MethodBodySemanticModel.cs

```diff
@@ -224,6 +225,13 @@ internal override bool TryGetSpeculativeSemanticModelCore(SyntaxTreeSemanticMode
 
             var methodSymbol = (MethodSymbol)this.MemberSymbol;
             binder = new WithNullableContextBinder(SyntaxTree, position, binder);
+
+            // PROTOTYPE(semi-auto-props): We should traverse until we get a property accessor, similar to Binder_Expressions. We may want to share the code.
```

Comment:
>// PROTOTYPE(semi-auto-props): We should traverse until we get a property accessor, similar to Binder_Expressions. We may want to share the code. [](http://example.com/codeflow?start=12&length=145)

I don't think we need to do any traversal here (a test will help to confirm that) because only type memberscan have ```MethodBodySemanticModel```, lambdas or local functions cannot have it.